src/structures/Message.js

```diff
@@ -448,6 +448,22 @@ class Message extends Base {
     });
   }
 
+  /**
+   * Publishes a message in an announcement channel to all channels following it.
+   * @returns {Promise<Message>}
+   * @example
+   * // Crosspost a message
+   * if (message.channel.type === 'news') {
+   *   message.crosspost()
+   *     .then(msg => console.log('Crossposted message'))
```

Comment:
```suggestion
   *     .then(() => console.log('Crossposted message'))
```
I believe that unused parameter shouldn't be there